packages/InteractiveIcon/svgs/NavButton/navButtonText.js

```diff
@@ -0,0 +1,75 @@
+export const cartEmptyBoldCopyDictionary = {
+  en: {
+    a11yText: 'Cart',
+  },
+  fr: {
+    a11yText: 'Panier',
+  },
+}
+
+export const cartFilledBoldCopyDictionary = {
+  en: {
+    a11yText: {
+      single: '%{numItems} item in cart',
+      multiple: '%{numItems} items in cart',
+    },
+  },
+  fr: {
+    single: '%{numItems} article dans le panier',
+    multiple: '%{numItems} articles dans le panier',
```

Comment:
These two keys should be nested in an `a11yText` object since **CartFilledBold** expects `a11yText.single` and `a11yText.multiple` for both English and French.